code/parachain/runtime/common/src/xcmp.rs

```diff
@@ -1,16 +1,15 @@
 //! proposed shared XCM setup parameters and impl
```

Comment:
opinionated. these are rewards for block authors.

Reply:
neither it is about fees. one may but would not do as nobody does. please use common sense.

Reply:
on can make fees 0 - it is not about fees there. @vimukthi-git please stop doing https://en.wikipedia.org/wiki/List_of_fallacies#Red_herring_fallacies in review :)

Reply:
> please stop doing https://en.wikipedia.org/wiki/List_of_fallacies in review

Don't see the relevance. Naming things non-ambiguously is not a black art to argue about.

Reply:
why? it was named impls.rs . sure it was bad.

Reply:
It's ok to have aggregations of code(impl.rs or whatever) until we have a good understanding about the overall product/repo/concepts. i.e We will refactor runtime level code with a good design(RFC) when we can think about it better with production experience.

Reply:
so you think `impls.rs` is better file name then `rewards.rs`? what impls is? rewards is clear - fees to block authors. if you not allow fees to block authors (which is weird) and rename it to anything else, e.g. treasury.rs

Reply:
i named the way i assume better to audit. specifically fees.rs must be audited for XCM(and that way BYOG) setup, while rewards.rs is not touching that area.

Reply:
> Don't see the relevance. Naming things non-ambiguously is not a black art to argue about. tbh I think you tend to get into this type of scenarios because you are over-refactoring code, then more names to give :)

please point me to overrefactoring? specific line of code?

Reply:
impls.rs bad no matter what for review and audit. you can set final name to what ever what in next pr or rfc as you wish.

Reply:
> so you think impls.rs is better file name then rewards.rs? what impls is?

No, just saying don't refactor(leave things as is) or give a different name that matches the function.

Reply:
new name to matches functions. old name distracts (should one look into it when audit fees or XCM - no, but it impls something? - so it is just bad). fees protect from spam. where is spam protection? here. 

imagine gov, it has taxes === fees. and it has budges = rewards. budges is from fees but != fees.